src/app/static/src/js/components/reportLog/runningReportDetails.vue

```diff
@@ -2,6 +2,20 @@
     <div>
         <div v-if="reportLog">
             <div id="report-log">
+                <div class="row pt-2">
+                    <div id="report-name" class="col-sm-auto ">
+                        <span>Report:</span>
+                        <span class="font-weight-bold">
+                            {{ reportLog.report }}
+                        </span>
+                    </div>
+                    <div id="report-start" class="col-sm-auto ">
+                        <span>Run started:</span>
+                        <span class="font-weight-bold">
+                            {{ formattedReportDate }}
+                        </span>
+                    </div>
+                </div>
```

Comment:
It's probably due to my setup but the report details aren't showing for me. I just get the message:
```
There are no logs to display
Error: An unexpected error occurred. Please contact support at montagu-help@imperial.ac.uk and quote this code: uvg-mrz-qfw
```

Reply:
Do you actually have any report runs in your db? Or can you see and select them in the list, but the details just aren't showing when you select one?

Reply:
The latter.

Reply:
This get request is failing in all my builds now:
![image](https://user-images.githubusercontent.com/57633087/115707342-919e6280-a366-11eb-90ae-75e867d7b26b.png)

Reply:
Is there an error and stack trace for that 500 showing in the backend?

Reply:
`[qtp1519873045-17] WARN org.vaccineimpact.orderlyweb.app_start.ErrorHandler - For request /running/subcrystalline_iberianmidwifetoad/logs/, a UnexpectedError occurred with the following problems: [An unexpected error occurred. Please contact support at montagu-help@imperial.ac.uk and quote this code: ufx-jgr-fuh]
[qtp1519873045-31] WARN org.vaccineimpact.orderlyweb.app_start.ActionResolver - Exception was thrown whilst using reflection to invoke class org.vaccineimpact.orderlyweb.controllers.web.ReportRunController.getRunningReportLogs, see below for details
[qtp1519873045-31] ERROR org.vaccineimpact.orderlyweb.app_start.ErrorHandler - An unhandled exception occurred
org.jooq.exception.DataAccessException: SQL [update orderlyweb_report_run set status = ?, report_version = ?, logs = ? where orderlyweb_report_run."key" = ?]; [SQLITE_CONSTRAINT]  Abort due to constraint violation (FOREIGN KEY constraint failed)
        at org.jooq_3.13.5.SQLITE.debug(Unknown Source)
        at org.jooq.impl.Tools.translate(Tools.java:2753)
        at org.jooq.impl.DefaultExecuteContext.sqlException(DefaultExecuteContext.java:755)
        at org.jooq.impl.AbstractQuery.execute(AbstractQuery.java:385)
        at org.jooq.impl.AbstractDelegatingQuery.execute(AbstractDelegatingQuery.java:119)
        at org.vaccineimpact.orderlyweb.db.repositories.OrderlyWebReportRunRepository.updateReportRun(ReportRunRepository.kt:139)
        at org.vaccineimpact.orderlyweb.controllers.web.ReportRunController.getRunningReportLogs(ReportRunController.kt:36)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.vaccineimpact.orderlyweb.app_start.ActionResolver.invokeControllerAction(ActionResolver.kt:29)
        at org.vaccineimpact.orderlyweb.app_start.Router$getWrappedRoute$1.handle(Router.kt:104)
        at spark.ResponseTransformerRouteImpl$1.handle(ResponseTransformerRouteImpl.java:47)
        at spark.http.matching.Routes.execute(Routes.java:61)
        at spark.http.matching.MatcherFilter.doFilter(MatcherFilter.java:130)
        at spark.embeddedserver.jetty.JettyHandler.doHandle(JettyHandler.java:50)
        at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1568)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
        at org.eclipse.jetty.server.Server.handle(Server.java:530)
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:347)
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:256)
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:102)
        at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:247)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:140)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:131)
        at org.eclipse.jetty.util.thread.ReservedThreadExecutor$ReservedThread.run(ReservedThreadExecutor.java:382)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:708)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:626)
        at java.lang.Thread.run(Thread.java:748)
Caused by: org.sqlite.SQLiteException: [SQLITE_CONSTRAINT]  Abort due to constraint violation (FOREIGN KEY constraint failed)
        at org.sqlite.core.DB.newSQLException(DB.java:941)
        at org.sqlite.core.DB.newSQLException(DB.java:953)
        at org.sqlite.core.DB.execute(DB.java:854)
        at org.sqlite.jdbc3.JDBC3PreparedStatement.execute(JDBC3PreparedStatement.java:56)
        at org.jooq.tools.jdbc.DefaultPreparedStatement.execute(DefaultPreparedStatement.java:209)
        at org.jooq.impl.AbstractQuery.execute(AbstractQuery.java:453)
        at org.jooq.impl.AbstractDMLQuery.execute(AbstractDMLQuery.java:904)
        at org.jooq.impl.AbstractQuery.execute(AbstractQuery.java:371)
        ... 29 more
[qtp1519873045-31] WARN org.vaccineimpact.orderlyweb.app_start.ErrorHandler - For request /running/subcrystalline_iberianmidwifetoad/logs/, a UnexpectedError occurred with the following problems: [An unexpected error occurred. Please contact support at montagu-help@imperial.ac.uk and quote this code: uie-bcb-ide]`

Reply:
The error seems to have resolved itself and I can no longer replicate it. Hopefully just a one off.

Reply:
It seemed to be having problems updating the report run table, with an FK violation - possibly because of the report version not existing. Might be a git vs demo issue?